cmd/titus-executor/main.go

```diff
@@ -89,6 +95,11 @@ func mainWithError(c *cli.Context, dockerCfg *docker.Config, cfg *config.Config)
 		m = metrics.Discard
 	default:
 		m = metrics.New(ctx, log.StandardLogger(), tag.Defaults)
+		if enableDefaultTitusTags {
+			// If fast property titus.executor.enableDefaultTitusTags is unset or set to true,
+			// wrap metrics.Reporter with runner.reporter that has default Atlas tags (t.jobId, t.taskId)
+			m = runner.NewReporter(m)
+		}
```

Comment:
that m implements the tagger interface is enough for the Runner to tell if extra Atlas tags is needed.  We don't need to pass any extra arguments into runner.New()